seldon/values.yaml

```diff
@@ -0,0 +1,39 @@
+ambassador:
```

Comment:
is this file must be maintained here?

Reply:
Its a file for settings to convert the core Seldon helm chart to kustomize so future updates can use it.